src/instruction_interpreter.py

```diff
@@ -71,8 +71,36 @@ def interpret_group_0(self, instruction):
             logger.warning(f"OpCode {instruction:X} not supported!")
 
     def jump(self, instruction):
+        """
+        1nnn - JP addr
+        Jump to location nnn.
+
+        The interpreter sets the program counter to nnn.
+        """
         self.program_counter = instruction & 0x0FFF
 
+    def set_vx_to_kk(self, instruction):
+        """
+        6xkk - LD Vx, byte
+        Set Vx = kk.
+
+        The interpreter puts the value kk into register Vx.
+        """
+        x = (instruction & 0x0F00) >> 8
+        kk = instruction & 0x00FF
+        self.reg_v[x] = kk
+
+    def add_kk_to_vx(self, instruction):
+        """
+        7xkk - ADD Vx, byte
+        Set Vx = Vx + kk.
+
+        Adds the value kk to the value of register Vx, then stores the result in Vx.
+        """
+        x = (instruction & 0x0F00) >> 8
+        kk = instruction & 0x00FF
+        self.reg_v[x] += kk
```

Comment:
Should probably do something like:
self.reg_v[x] = (self.reg_v[x] + kk) % 0x100